src/client/app/app.component.ts

```diff
@@ -1,11 +1,9 @@
 import { Component } from '@angular/core';
 import { ROUTER_DIRECTIVES, Routes } from '@angular/router';
 
-import { AboutComponent } from '../+about/index';
-import { HomeComponent } from '../+home/index';
-import { NameListService } from '../shared/index';
-import { NavbarComponent } from './navbar.component';
-import { ToolbarComponent } from './toolbar.component';
+import { AboutComponent } from './+about';
```

Comment:
We need to change `tsconfig.json`. for now `import * as foo from '../shared/index'` is fine.